cmd/ak/cmd/connections/create.go

```diff
@@ -56,6 +59,21 @@ var createCmd = common.StandardCommand(&cobra.Command{
 			return fmt.Errorf("create connection: %w", err)
 		}
 
+		if !quiet {
+			conn, err := connections().Get(ctx, cid)
+			if err != nil {
+				return fmt.Errorf("get connection: %w", err)
+			}
+
+			if l := conn.Links().InitURL(); l != "" {
+				action := "and can be initialized"
+				if conn.Capabilities().RequiresConnectionInit() {
+					action = "but requires initialization"
+				}
+				fmt.Fprintf(cmd.ErrOrStderr(), "Connection created, %s. Please run `ak connection init %v` to complete.\n", action, cid)
```

Comment:
Suggestion, since `` characters are not as useful in the terminal as in markdown.

```suggestion
				fmt.Fprintf(cmd.ErrOrStderr(), "Connection created, %s. Please run this to complete: ak connection init %v\n", action, cid)
```